ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -840,6 +841,14 @@ import '../nuxeo-button-styles.js';
       }
     }
 
+    _onCheckBoxKeydown(e) {
+      // check for enter or space
+      if (e.keyCode === 13 || e.keyCode === 32) {
+        // this.$.checkmark._tap();
```

Comment:
Can we have a comment on why we're keeping this?

Reply:
➕ 1

Reply:
This is a leftover, not supposed to be here. I will delete it 👍